### PR TITLE
appveyor: Update MSVC to 2015 and Qt to 5.8.0

### PR DESCRIPTION
**Pull request by IceN9ne**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ install:
 build_script:
     - mkdir build
     - cd build
-    - '"c:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/vcvarsall" %vc_arch%'
+    - '"c:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall" %vc_arch%'
     - path
     - ps: |
         $zlibinc = c:\cygwin\bin\find /cygdrive/c/deps/ -path '*v120*/zlib.h'
```
Comment (DarthGandalf) [inline .appveyor.yml]: You might need to change this to something like v140

Comment (IceN9ne) [reply .appveyor.yml]: Is that what the `v120` stands for?  I assumed it was version 1.2.0 of `zlib`.

Comment (DarthGandalf) [reply .appveyor.yml]: Yeh, this is an unsupported way to use nuget, the proper one is through visual studio GUI; I didn't know what is the format of filenames there, but needed to select a single file, without hardcoding too much.

Maybe there is a better way to use nuget from cmake-generated nmake...

**Comment by Stanzilla** [comment]
do it!

**Comment by ctrlaltca** [comment]
As a side note, Qt 5.8 is quite broken under osx and we had to revert using 5.7 in [another project](https://github.com/Cockatrice/Cockatrice/issues/2647).

**Pull request merged**